src/wristpy/processing/calibration.py

```diff
@@ -417,6 +433,59 @@ def _closest_point_fit(self, no_motion_data: np.ndarray) -> LinearTransformation
 
         return LinearTransformation(scale=scale, offset=offset)
 
+    def _closest_point_fit_gradient_descent(
+        self, no_motion_data: np.ndarray
+    ) -> LinearTransformation:
+        """Find linear transformation parameters that minimzes distance to unit sphere.
+
+        This function implements the scipy optimize.minimze function to find the
+        optimial scale and offset parameters that will minimze the error function,
+        which is defined as the distance of the no_motion acceleration data
+        from the unit sphere (where the unit sphere represents the ideal data points
+        under 1g acceleartion due to Earth's gravity). The initial guess for the
+        scale and offset are chosen by the equivalent transformation that would z-score
+        the no_motion_data.
+
```

Comment:
I'd explicitly document the reason for the bounds, as this may look odd to an uninformed reader.